spec/features/work_packages/select_query_spec.rb

```diff
@@ -37,11 +37,17 @@
                               member_through_role: role
   }
 
-  let(:filter_name) { 'done_ratio' }
-  let(:i18n_filter_name) { WorkPackage.human_attribute_name(filter_name.to_sym) }
+  let(:filter_1_name) { 'assigned_to_id' }
+  let(:filter_2_name) { 'done_ratio' }
+  let(:i18n_filter_1_name) { WorkPackage.human_attribute_name(filter_1_name.to_sym) }
+  let(:i18n_filter_2_name) { WorkPackage.human_attribute_name(filter_2_name.to_sym) }
+
   let!(:query) do
     query = FactoryGirl.build(:query, project: project, is_public: true)
-    query.filters = [Queries::WorkPackages::Filter.new(filter_name, operator: '>=', values: [10])]
+    query.filters = [
+      Queries::WorkPackages::Filter.new(filter_1_name, operator: '=',  values: ['me']),
+      Queries::WorkPackages::Filter.new(filter_2_name, operator: '>=', values: [10])
+    ]
     query.save and return query
```

Comment:
@0xF013 yep, I wouldn't have thought to do that with `.tap`.. `query.save and query` would actually suffice here as well.